src/scenarios/blazor/pre.py

```diff
@@ -1,14 +1,15 @@
 '''
 pre-command
 '''
-import sys, os
+import sys, os, subprocess
 from performance.logger import setup_loggers
 from shared import const
 from shared.precommands import PreCommands
 from test import EXENAME
 
 # For blazor3.2 the linker argument '--dump-dependencies' should be added statically to enable linker dump
 # For blazor5.0 the linker argument can be added to the command line as an msbuild property
+subprocess.run(["dotnet", "workload", "uninstall", "microsoft-net-sdk-blazorwebassembly-aot"])
```

Comment:
Why do we need to uninstall the workload before installing wasm-tools workload later?

Reply:
For the non-AOT runs I want this toolset removed because it seems to be causing weird behavior, where the runs will try and AOT the builds even though that is not specified in the csproj. It also just seems like good hygiene to make sure that we have these thing uninstalled after we are done using them.

Reply:
And this way we can make sure that all the machines have them cleaned off before we ever do any runs. Once we have the machines back in a good state we can take these out and only run the post step.

Reply:
Sounds good to me